### ISSUE TITLE
Cloning pulls 1.5.9-RC2 installer instead of latest stable 1.5.8

### ISSUE THREAD

**Issue by Murlkk** [FOGProject/fogproject#394]
Hey Guys,

Linux/FOG/GitHub beginner here and i am not sure if i am doing something wrong when getting ready to install FOG. Running the below command seems to clone the 1.5.9-RC2 Installer/Updater instead of the 1.5.8 latest stable version that i see listed in the releases section.
`git clone https://fogproject/fogproject.git fogproject-master`

When i setup my previous FOG server a couple months ago i ran the same command and downloaded the 1.5.8 installer just fine and got everything installed and working properly. This server is going to be for a remote office so i would like to get the same stable version that i have been using successfully here at my location if possible.

Love the FOGProject so far! It's a huge quality of life improvement from our previous non-existent imaging solution

**Comment by Sebastian-Roth** [comment]
@Murlkk Not sure if I get what you mean with this. Let me try to explain things a bit.

If you clone the github repository you will always have the default branch checked out at first (unless you specify parameters). We have the default set to a branch called `master` which points to the latest stable version of FOG - 1.5.8 as of now. If you really want to use the latest development version you run `git checkout dev-branch` after the initial git clone and you can install FOG 1.5.9-RC2 from that.

By the way, the `git clone https://fogproject/fogproject.git fogproject-master` command is probably not going to work at all! Should be `git clone https://github.com/FOGproject/fogproject.git fogproject-master`.

**Comment by Murlkk** [comment]
You are totally right about the command i wrote above haha! Like i said total beginner here :)
Went ahead and deleted the previous clone folder and started again and still get 1.5.9-RC2 so not sure what i am doing wrong, my commands in red:
![image](https://user-images.githubusercontent.com/67655705/86178483-4700e080-badd-11ea-9311-b13b897b5571.png)

**Comment by Sebastian-Roth** [comment]
@Murlkk Ohh well. Got me there. I've had way too little sleep over the last weeks and working on other projects a lot so I got lost with this. Forget what I said above. To encourage more people to use the release candidate (RC) versions so we can get more feedback from users before the final release we decided to merge the RC code into the master branch for official RC releases. So sure, current `master` really is 1.5.9-RC2!

I would encourage you to also use 1.5.9-RC2 and give us feedback on this. It has a keen amount of bugs fixed compared to 1.5.8. But if you are still keen to get the old stable 1.5.8 from github you need to run the following commands:

```
git clone https://github.com/FOGproject/fogproject.git fogproject-master
cd fogproject-master
git checkout 1.5.8
cd bin
./installfog.sh
```

To switch back to master run `git checkout master` in the project directory.

**Comment by Murlkk** [comment]
If it wasn't a remote office i would totally give the RC a go. Your latest commands worked perfectly so i am up and running! Thanks for the quick responses
